include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -922,7 +922,7 @@ def SOp : OneTargetOp<"s"> {
 def SwapOp : TwoTargetOp<"swap", [Hermitian]> {
   let summary = "Swap operation";
   let description = [{
-    This operation swaps the states of two wires.
+    This operation swaps the states of two qubits.
```

Comment:
Another terminology gotcha. I think the definitions here in the tablegen files meticulously _avoid_ using the term qubit since Quake doesn't define qubit and has no type qubit, etc.

Reply:
Measurement, sink, and u3 also mention qubits. The phrasing "swaps the states of two wires" is simply unclear to me - what is the state of a wire? I can undo this (I made the change when I thought I could pick up generated docs), but I think we need to be really clear about what a wire is and not use state, qubit, and wire synonymously.

Reply:
Yeah, I overstated the case. I should've said some attempts were made to clean up the documentation and to avoid qubit and some other terms to raise the level of abstraction away from words that can/do lead to confusion. Those cleanup passes, no doubt, missed things. Agreed that we should not be conflating terms and should be consistent, which was really my initial point. At one point "wire" was considered the better term than "qubit". Maybe that needs to be revisited?